dev/NavigationView/NavigationView.cpp

```diff
@@ -293,6 +312,18 @@ void NavigationView::OnApplyTemplate()
 
     m_contentLeftPadding.set(GetTemplateChildT<winrt::FrameworkElement>(c_contentLeftPadding, controlProtected));
 
+    m_paneHeaderCloseButtonColumn.set(GetTemplateChildT<winrt::ColumnDefinition>(c_paneHeaderCloseButtonColumn, controlProtected));
```

Comment:
>m_paneHeaderCloseButtonColumn.set(GetTemplateChildT<winrt::ColumnDefinition>(c_paneHeaderCloseButtonColumn, controlProtected)); [](start = 4, length = 127)

Why do these template parts get different treatment than the template parts above?